### PR TITLE
SNOW-1732002: Add disable cte tests

### PR DESCRIPTION
**Pull request by sfc-gh-jdu**
1. Which Jira issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   

   Fixes SNOW-1732002

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
      - [ ] If this test skips Local Testing mode, I'm requesting review from @snowflakedb/local-testing
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency
   - [ ] If this is a new feature/behavior, I'm adding the Local Testing parity changes.

3. Please describe how your code solves the related issue.

   Because parameters for cte is enabled by default on the test account now, we can flip the test

### CODE REVIEW

File: tests/unit/test_open_telemetry.py [hunk 1]
```diff
@@ -15,10 +15,6 @@
 
 pytestmark = [
     pytest.mark.udf,
-    pytest.mark.skipif(
```
Comment (sfc-gh-jdu) [inline tests/unit/test_open_telemetry.py:18]: confirmed with @sfc-gh-yuwang that it can be removed now.

**Review by sfc-gh-jrose — APPROVED** [review]

**Comment by sfc-gh-yzou** [comment]
@sfc-gh-jdu will you be able to open a daily test with this, or we have to test after merge

**Review by sfc-gh-yzou — APPROVED** [review]

**Comment by sfc-gh-jdu** [comment]
> @sfc-gh-jdu will you be able to open a daily test with this, or we have to test after merge

yes, running it right now

**Review by sfc-gh-helmeleegy — APPROVED** [review]
lgtm, thanks!

**Pull request merged**